lib/ht_sip_validator/validator/ocr/well_formed_xml.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+require "nokogiri"
+
+# frozen_string_literal: true
+module HathiTrust::Validator
+
+  # Validates that provided coordinate OCR is UTF-8
+  class OCR::WellFormedXML < FileValidator
+
+    def perform_file_validation(filename, filehandle)
+      check_well_formed_xml(filename, filehandle)
+    end
+
+    def should_validate?(filename)
+      !filename.match(/\.(html|xml)$/).nil?
+    end
+
+    private
+
+    def check_well_formed_xml(filename, filehandle)
+      messages = []
+      begin
+        Nokogiri::XML(filehandle) do |config|
```

Comment:
Should there also be a corresponding check_well_formed_html method that uses `Nokogiri::HTML` ?